comms/Cargo.toml

```diff
@@ -65,6 +65,6 @@ tempfile = "3.1.0"
 tari_common = { version = "^0.10", path = "../common", features = ["build"] }
 
 [features]
-bundled_openssl = ["openssl-sys/vendored"]
+c_integration = []
```

Comment:
now is this correct? All the other clients of tari_comms  would like to bundle openssl but we are removing that option because just the FFI crate doesn't like it. The FFI crate should just not use this feature.

The problem of course is that Core and the Wallet crates use Comms and they both by default use the bundled features in comms. So we should make a feature in Core and Wallet that excludes this feature from Comms. That way Wallet_FFI can choose to use non-default features to exclude this bundling rather than making the default to exclude it.